concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,184 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Clone concent repositories
+          git:
+            repo:    "{{ item.url }}"
+            dest:    "{{ jenkins_home_dir }}/{{ item.name }}"
+            clone:   yes
+            update:  yes
+            version: dev
+            force:   yes
+          with_items:
+          #  - { url: "{{ repositories['concent-deployment'].url }}", name: concent-deployment }
+            - { url: "https://github.com/bartoszbetka/concent-deployment.git", name: concent-deployment }
+            - { url: "{{ deployment_values_repository }}", name: concent-deployment-values }
+
+        - name: Create directory for concent jenkins secrets
+          file:
+            path:  "{{ jenkins_home_dir }}/{{ item }}"
+            state: directory
+            owner: jenkins
+            group: jenkins
+          with_items:
+            - concent-secrets
+            - concent-secrets/{{ cluster }}/
+            - concent-secrets/cloud/
+            - concent-secrets/jenkins/
+
+    - become:      yes
+      become_user: root
+      block:
+        - name: Upload secrets
+          copy:
+            src:   "{{ local_secret_dir }}/{{ cluster }}/{{ item }}"
+            dest:  "{{ jenkins_home_dir }}/concent-secrets/{{ cluster }}/{{ item }}"
```

Comment:
Do we really need to copy the secrets to Jenkins directory? Can't it use the ones from `/var/`?

Reply:
This is question for separate issue and changes deployment process.

Reply:
Why though? You're copying them here. It's not a part of a different playbook.

Reply:
The solution from your comment is not that obvious. Do we want assume that `/var/concent-data` path is always exist ? That may work, but we must run the `cluster-deploy-secrets.yml` playbook in specific case.  I think that the `configure-jenkins.yml` playbook should be independent.

Reply:
It's fine to assume that `/var/concent-data/`, but you're right - I don't want to require running `cluster-deploy-secrets.yml`. We need a separate playbook. But in that case let's leave it as is for now. We can add that playbook later.